plugins/module_utils/_api_data.py

```diff
@@ -1700,6 +1700,127 @@ def join_path(path):
             },
         ),
     ),
+    ('interface', 'wifiwave2'): APIData(
+        unversioned=VersionedAPIData(
```

Comment:
I didn't add versioning because in worst case it will just won't work on the device, so I thought people can figure out whether `wifiwave2` package is installed or not (compared to when depending on version you may get different required parameters or behavior).
But I think there is no way to specify both min and max version (i.e. range) in the `_api_data`, right?

Reply:
If I understood you correctly in one of your previous comments we currently cannot say e.g. "command `asdf` was introduced in version 7.10 but renamed to `dfgh` in version 7.12". So the command `asdf` would only be valid for version between and including 7.10 and 7.11.x.
@felixfontein Do you see a possibility to enhance the already extremely useful feature to have commands version-specific to have a range of versions?

Reply:
Right now it's not, but I created a PR that would allow this: #238

Basically you can do things like:
```py
    ('interface', 'wifiwave2'): APIData(
        versioned=[
            ('7.8.3', '==', 'For some reason RouterOS 7.8.3 did not suppor this'),
            ('7.10', '>=', 'the interface wifiwave2 path has been renamed to foo bar in RouterOS 7.10'),
            ('7.8', '>=', VersionedAPIData(...)),
        ],
    ),
```
I think this would be helpful? (The `versioned` entries are processed from top to buttom.)

Reply:
Wow, that's amazing! Thank you.

For this very specific case (wifiwave2) I have to say that I unfortunately don't have a device to test it on =/

Reply:
I've merged the PR, please rebase :)

Reply:
Will do. A bit busy this week 😔.
So you want me to squash the changes or is it better to do it with the merge?

Reply:
We squash on merge anyway, so no need to squash them. (Sometimes there are PRs which end up with 50+ commits, there cleaning up even before squash-merging makes sense, but this one is faaar from that ;) )